OutOfSchool/OutOfSchool.WebApi/Models/ProvidersInfo/CodeficatorAddressWorkshopInfoDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OutOfSchool.WebApi.Models.ProvidersInfo;
+
+public class CodeficatorAddressWorkshopInfoDto
+{
+    public int Id { get; set; }
+
+    public string Category { get; set; }
```

Comment:
Could you explain this `string Category` prop and why it is not present in other `CodeficatorAddressProviderInfoDto`?